ontology/obib.md

```diff
@@ -3,10 +3,9 @@ layout: ontology_detail
 id: obib
 title: Ontology for Biobanking
 contact:
-  email: jiezheng@pennmedicine.upenn.edu
-  github: zhengj2007
-  label: Jie Zheng
-  orcid: 0000-0002-2999-0103
+  email: jmwhorton@uams.edu
+  github: jmwhorton
+  label: Justin Whorton
```

Comment:
@jmwhorton needs a valid orcid

Reply:
Updated and added orcid.  (Apologies, I didn't have one yet.)